src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -419,6 +419,46 @@ public async Task TestOpenSolution_LoadMetadataForReferencedProjects()
             }
         }
 
+        [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
+        [WorkItem(33047, "https://github.com/dotnet/roslyn/issues/33047")]
+        public async Task TestOpenProject_CSharp_GlobalPropertyDesignTimeBuildDefault()
```

Comment:
Can you add a test for passing in a properties dictionary per https://github.com/dotnet/roslyn/issues/33047?

Reply:
the other new test below passes in a properties dictionary, but with a non-default value for DesignTimeBuild and not ShouldUnsetParentConfigurationAndPlatform (as per #33047). both properties are specified in ProjectBuildManager.s_defaultGlobalProperties. i could change the test from DesignTimeBuild to ShouldUnsetParentConfigurationAndPlatform to exactly match the issue, but probably not worth in testing both?